examples/support/triangle/main.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2017 The Gfx-rs Developers.
```

Comment:
I'm assuming because it didn't exist before hand. I rewrote the basic triangle example using the support module, so users could compare implementing a basic triangle with and without the support module.